Userland/Libraries/LibELF/DynamicLinker.cpp

```diff
@@ -41,6 +41,8 @@ using LibCExitFunction = void (*)(int);
 using DlIteratePhdrCallbackFunction = int (*)(struct dl_phdr_info*, size_t, void*);
 using DlIteratePhdrFunction = int (*)(DlIteratePhdrCallbackFunction, void*);
 
+extern "C" NO_RETURN int _invoke_entry(int argc, char** argv, char** envp, EntryPointFunction entry);
```

Comment:
We don't use the `NO_RETURN` macro anywhere else, since `[[noreturn]]` is already a standard C++ attribute.

Sorry for the minor nitpick.

Reply:
Fixed. :)